src/entries/popup/pages/settings/customChain/index.tsx

```diff
@@ -277,6 +277,8 @@ const KNOWN_NETWORKS = {
   ],
 };
 
+const parseURL = (url: string | undefined) =>
+  url && (url.startsWith('http') ? url : `https://${url}`);
```

Comment:
If a user enters `localhost:8545` you'll autocomplete it to https://localhost:8545  which won't work so we have 2 options:

- Drop this and not add the prefix
- Add an exception for hosts `localhost` & `127.0.0.1` where you only add `http://`

Reply:
I'm removing it for now so I can merge it